.travis.yml

```diff
@@ -18,6 +18,7 @@ addons:
       - elasticsearch
       - openjdk-8-jre-headless
   postgresql: "9.6"
+  chrome: stable
```

Comment:
I think we need to update our `running tests` section with instruction for set up required for running selenium tests. Can you please send a separate PR for that?

Reply:
I need to send a separate pr with the code that is required to run js tests?

Reply:
yes, please. not exactly code but instructions for packages to install. It would go [here](https://github.com/rubygems/rubygems.org/blob/master/CONTRIBUTING.md#environment-linux---debianubuntu).